atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/OracleConnectionConfig.java

```diff
@@ -196,16 +196,6 @@ protected final void check() {
         } else {
             Preconditions.checkArgument(!getTwoWaySsl(),
                     "two way ssl cannot be enabled without enabling ConnectionProtocol.TCPS");
-            Preconditions.checkArgument(!getServerDn().isPresent(),
-                    "a server dn cannot be given without enabling ConnectionProtocol.TCPS");
-            Preconditions.checkArgument(!getTruststorePath().isPresent(),
-                    "a truststore path cannot be given without enabling ConnectionProtocol.TCPS");
-            Preconditions.checkArgument(!getTruststorePassword().isPresent(),
-                    "a truststore password cannot be given without enabling ConnectionProtocol.TCPS");
-            Preconditions.checkArgument(!getKeystorePath().isPresent(),
-                    "a keystore file cannot be given without enabling ConnectionProtocol.TCPS");
-            Preconditions.checkArgument(!getKeystorePassword().isPresent(),
-                    "a keystore password without enabling ConnectionProtocol.TCPS");
```

Comment:
I'd prefer if we put a `WARN` log here if any of them are present? Basically it's easy to make a config error and then be really puzzled as to why the settings aren't taking effect.

Reply:
In this case if you set the protocol incorrectly you wouldn't be able to connect to the DB. Warning on here would just be unnecessary noise if the rest of the prefs were actually correct.